### ISSUE TITLE
Eng: Mechanical Dragonling Trinket Bug and possible Exploit.

### ISSUE THREAD

**Issue by ViperXSS2** [TeamEverlook/Everlook-BugTracker#363]
Hello

I made the low level Mechanical Dragonling Trinket and its interesting what happens when you use the Trinket in combat:    https://www.wowhead.com/classic/item=4396/mechanical-dragonling

When used, it will spawn the Dragonling, but your characters UI will break, not allowing you to click spells, in my case Vanish, or perform any attacks, potions ect. 

My Rogue will continue to auto attack: While the Dragonling fights.

After the fight, the Dragonling DE spawns (Dies), and should have a 1 Hour Cooldown to be re-used again. 

It however does not have a 1 hour cooldown, and can be used again instantly! The problem is since the UI doesn't function I cant click anything at all. So I usually have to re-log. 

Its a bug obviously, and re-using the Dragonling seems like an exploit while its broken so I have put my trinket in the bank for now sadly. 

It sucks because I went out of my way to make it early, right at lvl 30, so I could use it to lvl :( 

Thank you.

Bartab in game Rogue Dwarf Starfall

**Comment by jensbond420** [comment]
Yep, however you do need to relog to be able to re-use it. But indeed a bug, should be 1 hour Cooldown.

For the not allowing you to use spells, thats a WDB issue. Clear you WDB and try again. And If you get locked, hit escape twice.

**Comment by ViperXSS2** [comment]
Alright, I have switched over to the new everlook client. Its 1HR cooldown works, and it does not break my UI over here. It has however disappeared from my inventory somehow, but I still have it in my hotbar and can use it as its suppose to be used.
